.travis.yml

```diff
@@ -1,13 +1,25 @@
+language: ruby
+sudo: false
+cache: bundler
+
 rvm:
   - 2.1
-  - 2.2.2
-
-script: "bundle exec rake clean spec cucumber"
+  - 2.2
+  - 2.3
+  - 2.4
 
+# FIXME: fails with modern bundler
```

Comment:
Not sure modern bundler is the problem. I have locally tested with bundler 1.12.5 and I got the same errors.